galaxy/api/access.py

```diff
@@ -391,6 +391,22 @@ def can_delete(self, obj):
         return False
 
 
+class SessionEventAccess(BaseAccess):
+    model = models.SessionEvent
+
+    def can_read(self, obj):
+        return True
+
+    def can_add(self, data):
+        return True
+
+    def can_change(self, obj, data):
```

Comment:
maybe make it so that you can only change an object if the session ID's match

Reply:
Agree